templates/p5_gfd.bt

```diff
@@ -896,11 +910,7 @@ typedef struct
     f32 ClipPlaneFar;
     f32 FieldOfView;
     f32 AspectRatio;
-
-    if ( Version > 0x1104060 )
-    {
-        f32 Field190;
-    }
+    f32 Field190;
```

Comment:
why did you remove the version check

Reply:
answered on discord lol
i'll bring it back in next commit i guess